pyxrf/model/xrf_energy.py

```diff
@@ -166,11 +166,26 @@ def read_ref_data(ref_file_name):
         """Read file with reference data (for XANES fitting)"""
         ref_labels=[]
 
+        with open(ref_file_name, "r") as f:
```

Comment:
This is outdated code, I am not sure why it was offered for review. I changed the code so that it currently requires references in CSV files. Labels are parsed to a list, data to ndarray. I don't think it makes sense to bring in xarrays or pandas, since parsing is simple and there are library functions that do it without extra coding.